Design/sapflow_inventory.csv

```diff
@@ -0,0 +1,88 @@
+Tree_Code,Port,Species,Tag,PakBus,Grid_Square,Installation_Date
+C7,1,ACRU,1439,41,B3,2021-03-19
```

Comment:
This is duplicating information from the TEMPEST inventory file. Not the end of the world, just noting.